Packages/vcs/Lib/vtk_ui/manager.py

```diff
@@ -54,11 +70,24 @@ def queue_render(self):
     def elevate(self):
         # Raise to top layer of render window
```

Comment:
Added the `actor_renderer` to the stack as well, and made it so the layers get updated only if the manager's renderers aren't already on top.